README.md

```diff
@@ -63,6 +63,68 @@ Requests to cross-scheme subresources will no longer send `SameSite=Lax` or `Sam
 For example, if http://site.example embeds an image from https://site.example, `SameSite=Lax` and `SameSite=Strict` cookies will no longer be sent on the request to https://site.example, and any such cookies in the response will be ignored. 
 
 ## Questions
+### How does this affect iframes?
+Prior to "Schemeful Same-Site": if there are multiple nested iframes a given iframe is always cross-site, to the top-level frame, if any of its ancestors are cross-site, to the top-level frame.\
+With "Schemeful Same-Site": This can now occur if any ancestor iframe is cross-scheme, to the top-level frame.
+
+Any iframe using the `sandbox` attribute is always cross-site, to the top-level frame. Nothing here has changed.
+#### The src attribute
+A request to get the content specified by the `src` attribute is treated as any other subresource request and is affected the same ways.
+Once the iframe's content is loaded, it runs as cross-site or same-site depending the origin of the resulting document (this is typically the origin of the value of the `src` attribute).
+
+For example, the following request for the iframe's page will not be sent `SameSite=Lax` nor `SameSite=Strict` cookies (nor will its response be able to set those cookies) as it's cross-scheme. After the iframe's page is loaded, nothing on it is able to access those `SameSite` cookies because it's being run as cross-site.
+
+```
+<html>
+<body>
+
+<h1>This page is hosted at http://website.example/insecure.html</h1>
+<iframe src="https://website.example/secure.html" title="An iframe to a secure page"></iframe>
+
+</body>
+</html>
+```
+#### The srcdoc attribute
+Content that is loaded into an iframe through the `srcdoc` attribute inherits its parent's origin. Therefore it has the same "Schemeful Same-Site" restrictions (or lack thereof) as its parent document.
+
+### How does this affect scripts?
+#### Fetching scripts
+A request to get the script via the `src` attribute is treated as any other subresource request and is affected the same ways.
+
+For example, the following request for a script will not be sent `SameSite=Lax` nor `SameSite=Strict` cookies (nor will its response be able to set those cookies).
+```
+<html>
+<body>
+
+<h1>This page is hosted at http://website.example/insecure.html</h1>
+<script src="https://website.example/script-that-accesses-samesite-cookies.js"></script>
+
+</body>
+</html>
+```
+#### Executing scripts
+Scripts execute as part of their parent document, therefore they have the same "Schemeful Same-Site" restrictions (or lack thereof) as their parent document.
+
+This could affect websites that load scripts into a cross-scheme iframe. Such a script would be run as cross-site, due to the cross-scheme iframe, and would not have access to `SameSite=Lax` nor `SameSite=Strict` cookies.
```

Comment:
This is a bit un-intuitive. Wouldn't the script in the iframe also technically be same-site with the iframe document (its parent)?

Also, perhaps we should specify that you're talking about the `document.cookie` interface; because there might be other things like a `fetch` call to a cross-site URL where different rules would apply.

Reply:
The script is same-site with its parent (iframe) document, but it's ultimately cross-site with the top-level frame since the iframe is cross-site. Because the script cross-site with the top-level frame it's no longer allowed to access the SameSite cookies for that registrable domain.

However, I remembered as I typed this that it's a Chrome specific enforcement that Firefox doesn't follow. Not sure about Safari.

For your second point I'll clarify that I'm referring to `document.cookie`